pkg/sif/descriptor_input_test.go

```diff
@@ -185,6 +185,21 @@ func TestNewDescriptorInput(t *testing.T) {
 				OptSBOMMetadata(SBOMFormatCycloneDXJSON),
 			},
 		},
+		{
+			name: "OptOCIMediaType",
+			t:    DataOCIBlob,
+			opts: []DescriptorInputOpt{
+				OptOCIBlobMetadata("application/vnd.oci.image.config.v1+json"),
+			},
+		},
+		{
+			name: "OptOCIMediaTypeUnexpectedDataType",
+			t:    DataGeneric,
+			opts: []DescriptorInputOpt{
+				OptOCIBlobMetadata("application/vnd.oci.image.config.v1+json"),
+			},
+			wantErr: &unexpectedDataTypeError{DataGeneric, []DataType{DataOCIBlob}},
+		},
```

Comment:
Nit: `name` should match up with the option for consistency.

```suggestion
		{
			name: "OptOCIBlobMetadata",
			t:    DataOCIBlob,
			opts: []DescriptorInputOpt{
				OptOCIBlobMetadata("application/vnd.oci.image.config.v1+json"),
			},
		},
		{
			name: "OptOCIBlobMetadataUnexpectedDataType",
			t:    DataGeneric,
			opts: []DescriptorInputOpt{
				OptOCIBlobMetadata("application/vnd.oci.image.config.v1+json"),
			},
			wantErr: &unexpectedDataTypeError{DataGeneric, []DataType{DataOCIBlob}},
		},
```